core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -675,19 +676,27 @@ public FileInfo getFileInfo(long fileId)
   public FileInfo getFileInfo(AlluxioURI path, GetStatusOptions options)
       throws FileDoesNotExistException, InvalidPathException, AccessControlException {
     Metrics.GET_FILE_INFO_OPS.inc();
-
     try (JournalContext journalContext = createJournalContext();
+         AuditContext auditContext = createAuditContext();
          LockedInodePath inodePath = mInodeTree.lockInodePath(path, InodeTree.LockMode.READ)) {
-      mPermissionChecker.checkPermission(Mode.Bits.READ, inodePath);
+      checkAndAppendAuditLog("getFileInfo", path.toString(), null, auditContext);
```

Comment:
don't even call `toString` within the lock here because it is possible audit log is not enabled but you still convert path to String which is a waste.